composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewEvent.kt

```diff
@@ -0,0 +1,15 @@
+package com.majotyler.hiittimer.presentation
+
+import androidx.compose.runtime.Immutable
+
+@Immutable
+sealed interface TimerViewEvent {
+    /** The user has clicked a button to start the timer. */
+    data object ClickAdd : TimerViewEvent
+    data class onDelete(val item: String): TimerViewEvent
```

Comment:
Data class should be all capitals, and I recommend following the pattern of making these always "ClickedX" 

```suggestion
    data class ClickedDelete(val index: Int): TimerViewEvent
```